src/net/db.rs

```diff
@@ -456,21 +511,67 @@ impl PeerDB {
         Ok(())
     }
 
-    fn update_local_peer(
+    fn get_schema_version(conn: &Connection) -> Result<String, db_error> {
+        let version = conn
+            .query_row(
+                "SELECT MAX(version) from db_config",
+                rusqlite::NO_PARAMS,
+                |row| row.get(0),
+            )
+            .optional()?
+            .unwrap_or("1".to_string());
+        Ok(version)
+    }
+
+    fn apply_schema_2<'a>(tx: &Transaction<'a>) -> Result<(), db_error> {
+        debug!("Apply schema 2 to peer DB");
+        for row_text in PEERDB_SCHEMA_2 {
+            tx.execute_batch(row_text).map_err(db_error::SqliteError)?;
+        }
+        Ok(())
+    }
+
+    fn apply_schema_migrations<'a>(tx: &Transaction<'a>) -> Result<String, db_error> {
+        debug!("Apply any schema migrations");
```

Comment:
This debug line seems spurious. This will echo on every open of the PeerDB, correct? There's already other debug lines that echo whenever the PeerDB opens.

Reply:
Fixed